src/main/java/org/opentripplanner/updater/transit/ticket/AvailableTransitTicketsGetter.java

```diff
@@ -0,0 +1,45 @@
+package org.opentripplanner.updater.transit.ticket;
+
+import com.fasterxml.jackson.core.Version;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+import org.opentripplanner.pricing.transit.ticket.TransitTicket;
+import org.opentripplanner.updater.transit.ticket.deserializer.TransitTicketDeserializer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+
+public class AvailableTransitTicketsGetter {
+
+    private static final Logger LOG = LoggerFactory.getLogger(AvailableTransitTicketsGetter.class);
+
+    public Set<TransitTicket> getFromFile(String filename) {
+        Set<TransitTicket> transitTickets = null;
+
+        try {
+            String json = new String(Files.readAllBytes(Paths.get(filename)));
+            ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
Creating proper object mapper should be done in constructor, in this method we should only have reading file and deserializing it